clients/go/pkg/justification/processor.go

```diff
@@ -90,6 +92,22 @@ func (p *Processor) Process(ctx context.Context, logReq *api.AuditLogRequest) er
 	}
 
 	logReq.Payload.Metadata.Fields[LogMetadataKey] = structpb.NewStructValue(&tokStruct)
-	// TODO(#266): Also populate RequestAttribute.Reason.
+
+	justs, ok := (*tok).Get("justs")
```

Comment:
Do you want what's the actual type of `justs` (the returned type is `interface{}`) - the returned value.

Reply:
Maybe I didn't get the context. We need the encoded json of `justs` and `json.Marshal` accepts `interface{}`. Why do we care about the actual type of `justs`?

Reply:
What if it's already a json string? Also out of curiosity :)

Reply:
```
	bytes, err := json.Marshal(`[{"category":"explanation","value":"need-access"}]`)
			if err != nil {
				return
			}
			fmt.Println(string(bytes))
```
The format is the same

Reply:
I know the marshaling a string is still the same string. But I'm still curious what's the original type from the token. This is not blocking.

Reply:
Looks like the type is a list of `justification` https://github.com/abcxyz/jvs/blob/5ab0828c78626fee6284de86035be5ebb558565f/pkg/justification/processor.go#L160 and https://github.com/abcxyz/jvs/blob/5ab0828c78626fee6284de86035be5ebb558565f/client-lib/go/client/jvs_client_test.go#L238